spec/controllers/recordings_controller_spec.rb

```diff
@@ -115,26 +132,18 @@
   #     recording = create(:recording)
   #     expect { delete :destroy, params: { id: recording.id } }.to change(Recording, :count).by(-1)
   #   end
-
+  #   it 'admin without ManageRecordings permission cannot delete recording from the database' do
```

Comment:
I know that this irrelevant to this PR,
But, what's the reasoning behind these test group been commented?

Reply:
It was to do with the `BigBlueButtonApi.new.delete_recordings(record_ids: params[:id])` having to be moved out of the destroy method and be put in the Recording model in an `after_destroy`.

I think either way we would need to just mock the response of `BigBlueButtonApi.new.delete_recordings(record_ids: params[:id])`? @KH-Amir-TN

Reply:
That makes sense,
In such case IMO we'd like to have the `#destroy` tests that like any other API will assert the removal of recordings records in the DB while expecting certain responses for certain cases,
The action tests shouldn't make sense of the model `callbacks` but to avoid compromising the rigidity of the tests we'd stub the  `BigBlueButtonApi#delete_recordings` like you said,